test/toggleBlock.test.js

```diff
@@ -407,4 +407,37 @@ describe('ToggleBlock', () => {
       expect(defaultContentFromQuery).toBe(config.defaultContent);
     });
   });
+
+  describe('When a toggle and its nested blocks are copied and pasted', () => {
+    beforeEach(() => {
+      const blocks = generateFullToggle(toggleBlock, data);
+      blocks.forEach((block) => {
+        redactor.appendChild(block);
+      });
+    });
+
+    it('resets the duplicated ids from a toggle with 3 paragraphs', () => {
+      redactor.innerHTML += redactor.innerHTML;
+      const lastCopiedBlock = redactor.lastChild;
+      const { children } = redactor;
+      const index = children.length - 1;
+      resetIdToCopiedBlock(redactor, lastCopiedBlock, index, 3);
+
+      for (let i = 0; i < children.length - 4; i += 1) {
+        const originalBlock = children[i];
+        const copiedBlock = children[i + 4];
```

Comment:
@juliansg022 It seems like the `children.length - 4` and `children[i + 4]` code is related to the `3` integer which is the number of paragraphs... so I think we could create a variable for that, and use in those cases